spec/fixtures/converters/two_records.json

```diff
@@ -0,0 +1,5 @@
+[
+{ "ip": "93.184.216.34", "timestamp": "1629960621", "ports": [ {"port": 80, "proto": "tcp", "status": "open", "reason": "syn-ack", "ttl": 54} ] }
+,
+{ "ip": "93.184.216.34", "timestamp": "1629960621", "ports": [ {"port": 80, "proto": "tcp", "service": {"name": "title", "banner": "404 - Not Found"} } ] }
+]
```

Comment:
Maybe rename the file to `input.json`. `two_records.json` seems a bit too specific?